### ISSUE TITLE
Test for WCAG failure F43: using structural markup in a way that does not represent relationships in the content

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#157]
http://www.w3.org/WAI/GL/2013/WD-WCAG20-TECHS-20130711/F43

The objective of this technique is to describe a failure that occurs when structural markup is used to achieve a presentational effect, but indicates relationships that do not exist in the content. This is disorienting to users who are depending on those relationships to navigate the content or to understand the relationship of one piece of the content to another. Note that the use of HTML tables for layout is not an example of this failure as long as the layout table does not include improper structural markup such as `<th>` or `<caption>` elements.

 Failure Example 1: A heading used only for visual effect 
 Failure Example 3: Using blockquote elements to provide additional indentation
 Failure Example 4: Using the fieldset and legend elements to give a border to text
## Tests Procedure
1. Check that each element's semantic meaning is appropriate for the content of the element.
